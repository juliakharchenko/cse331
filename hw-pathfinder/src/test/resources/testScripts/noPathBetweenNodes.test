# Find a path with Dijkstra's algorithm when
# there is no path between two nodes.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C

FindPath simpleGraph A B