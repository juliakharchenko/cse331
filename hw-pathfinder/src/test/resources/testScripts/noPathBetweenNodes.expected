# Find a path with Dijkstra's algorithm when
# there is no path between two nodes.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph

path from A to B:
no path found
